java/grpc-client/src/main/java/io/vitess/client/grpc/GrpcClientFactory.java

```diff
@@ -268,22 +272,18 @@ private KeyStore loadKeyStore(final File keyStoreFile, String keyStorePassword)
       return null;
     }
 
-    final Enumeration<String> aliases;
     try {
-      aliases = keyStore.aliases();
-    } catch (KeyStoreException exc) {
-      return null;
-    }
-
-    while (aliases.hasMoreElements()) {
-      final String alias = aliases.nextElement();
-      final X509Certificate[] certCollection = loadCertCollectionForAlias(keyStore, alias);
-      if (certCollection != null) {
-        return certCollection;
+      PKIXParameters p = new PKIXParameters(keyStore);
+      X509Certificate[] certificates = p.getTrustAnchors().stream()
+          .map(TrustAnchor::getTrustedCert)
+          .toArray(X509Certificate[]::new);
+      if (certificates.length == 0) {
+        return null;
       }
+      return certificates;
+    }  catch (InvalidAlgorithmParameterException | KeyStoreException e) {
```

Comment:
Checkstyle is also complaining here how about `exc`

Reply:
Fixed the checkstyle issues, thanks.